index.bs

```diff
@@ -330,9 +330,8 @@ The {{pictureInPictureEnabled}} attribute's getter must return `true` if
 <a>allowed to use</a> the feature indicated by attribute name
 `picture-in-picture`, and `false` otherwise.
 
-<dfn>Picture-in-Picture support</dfn> is `true` if there is no
-previously-established user preference, restrictions, or platform limitation,
-and `false` otherwise.
+<dfn>Picture-in-Picture support</dfn> is `true` by default. It is `false` if
+there's a user preference that disables it or a platform limitation.
```

Comment:
Antother way to phrase this would be to say that "blah blah is false if there is no user preference that disables it or platform limitations. It is true otherwise."

Reply:
@mounirlamouri If https://github.com/WICG/picture-in-picture/pull/101/commits/3782973811f901e8b1425e5f5c5a1906c6167315 looks good, I'll merge.

Reply:
@mounirlamouri I've merged it.